enterprise/frontend/src/embedding/data-picker/saved-entity-picker/SavedEntityList.tsx

```diff
@@ -9,22 +9,20 @@ import { PERSONAL_COLLECTIONS } from "metabase/entities/collections/constants";
 import { PLUGIN_MODERATION } from "metabase/plugins";
 import { Box } from "metabase/ui";
 import { getQuestionVirtualTableId } from "metabase-lib/v1/metadata/utils/saved-questions";
-import type { CardType, Collection, DatabaseId } from "metabase-types/api";
+import type { Collection, DatabaseId } from "metabase-types/api";
 import { SortDirection } from "metabase-types/api/sorting";
 
 import SavedEntityListS from "./SavedEntityList.module.css";
 import { CARD_INFO } from "./constants";
 
 interface SavedEntityListProps {
-  type: CardType;
   selectedId: string;
   databaseId: DatabaseId;
   collection?: Collection;
   onSelect: (tableOrModelId: string) => void;
 }
 
 const SavedEntityList = ({
-  type,
```

Comment:
`type` can only be `model` now, so I hardcode it, please see the comment on `DataSelector.jsx`'s `getCardType()` function.